andela/hc-millennium-falcon
# #156577926 Allocate different stakeholders as owners of different HealthCheck alerts

#### What does this PR do?
Allows allocation of different stakeholders as owners of different HealthChecks

#### Description of Task to be completed?
A user who has team access to another user's healthchecks account can view and get notified on the status of all the jobs defined in that account. This PR helps to reduce the team member healthchecks scope and allow stakeholder management by assigning the team member the relevant checks.

![image](https://user-images.githubusercontent.com/26734746/40067446-d39bf706-586e-11e8-90b0-d0816f4e1c43.png)

#### How should this be manually tested?
A user with team access creates several checks and a team.
He or she invites a member to join the team and selects the checks the team member should access.
A team member can then login and view only the checks he or she has been assigned

#### Any background context you want to provide?
Currently team access guarantees all existing individual healthchecks jobs access. A user granting team access to their account should be able to assign different jobs to the person they are inviting to join the team. 

#### What are the relevant pivotal tracker stories?

[#156577926](https://www.pivotaltracker.com/story/show/156577926)
